api/src/db/repository/universe-property-repository.ts

```diff
@@ -0,0 +1,46 @@
+import { EntityManager, EntityRepository, In } from 'typeorm';
+
+import { UniverseProperty } from '../entities/universe-property';
+
+@EntityRepository()
+export class UniversePropertyRepository {
+  constructor(private manager: EntityManager) {}
+
+  async get(key: string) {
```

Comment:
I like convention where:
getXX is trying to get object and if object is missing throw an error 
findXX is trying to get oobject and if object is missing returns undefined

Reply:
https://github.com/alcarin-org/alcarin/pull/22/commits/bbffadd36c53a93fc33d73e9084371a85ce8591b

my proposition is to make all custom repositories extend default ones and use typeorm standard names. `findOne` and `findOneOrFail`: https://typeorm.io/#/entity-manager-api